source/deployment/sso-saml-adfs-msws2016.rst

```diff
@@ -0,0 +1,208 @@
+
+10. In the **Finish** screen, select the option **Configure claims issuance policy for this application**, and click **Close**. 
+
+	.. image:: ../../source/images/SSO-SAML-ADFS_add-new-relying-party-trust_011.png
+
+	You will now exit the configuration wizard and a **Claim Rules** editor opens.
+
+Create Claim Rules
+------------------
+
+1. In the **Issuance Transform Rules** tab of the **Claim Rules** editor, click the **Add Rule…** button.
+
+	.. image:: ../../source/images/SSO-SAML-ADFS_create-claim-rules_001.png
+
+	This action opens an **Add Transform Claim Rule Wizard**.
+
+2. In the **Choose Rule Type** screen, select **Send LDAP Attributes as Claims** from the drop-down menu, then click **Next**.
+
+	.. image:: ../../source/images/SSO-SAML-ADFS_create-claim-rules_002.png
+
+3. In the **Configure Claim Rule** screen, enter a **Claim Rule Name** of your choice, select **Active Directory** as the **Attribute Store** and add the following mapping:
+  - From the LDAP Attribute column, **select** ``E-Mail-Addresses``. From the Outgoing Claim Type, **type** ``Email``
+  - From the LDAP Attribute column, **select** ``Given-Name``. From the Outgoing Claim Type, **type** ``FirstName``
+  - From the LDAP Attribute column, **select** ``Surname``. From the Outgoing Claim Type, **type** ``LastName``
+  - From the LDAP Attribute column, **select** ``SAM-Account-Name``. From the Outgoing Claim Type, **type** ``Username``
+
+For Mattermost 3.4 and later, the *FirstName* and *LastName* attributes are optional.
+
+Then, click **Finish** to add the rule.
+
+Note that the entries in the **Outgoing Claim Type** column can be chosen to be something else. They can contain dashes but no spaces. Note that they will be used to map the corresponding fields in Mattermost later.
+
+	.. image:: ../../source/images/SSO-SAML-ADFS_create-claim-rules_003.png
+
+4. Create another new rule by clicking the **Add Rule** button.
+
+5. In the **Choose Rule Type** screen, select **Transform an Incoming Claim** from the drop-down menu, then click **Next**.
+
+	.. image:: ../../source/images/SSO-SAML-ADFS_create-claim-rules_004.png
+
+6. In the **Configure Claim Rule** screen, enter a **Claim Rule Name** of your choice, then
+  - Select *Name ID* for the **Incoming claim type**
+  - Select *Unspecified* for the **Incoming name ID format**
+  - Select *E-Mail Address* for the **Outgoing claim type**
+
+Moreover, select the **Pass through all claim values** option. Then click **Finish**.
+
+	.. image:: ../../source/images/SSO-SAML-ADFS_create-claim-rules_005.png
+
+7. Click **Finish** to create the claim rule, then **OK** to finish creating rules.
+
+8. Open Windows PowerShell as an administrator and run the following command:
+
+  ``Set-ADFSRelyingPartyTrust -TargetName <display-name> -SamlResponseSignature "MessageAndAssertion"``
+
+where <display-name> is the name you specified in step 6. In our example it would be ``mattermost``.
```

Comment:
```suggestion
where *<display-name>* is the name you specified in step 6. In our example it would be ``mattermost``.
```
Could you confirm that "step 6" refers to step 6 immediately preceding this? Or is it another step 6?

Reply:
Thanks for finding this, it was actually step 4 of Add a Relying Party Trust. Corrected it.